<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditQuestions</class>
 <widget class="QDialog" name="AddEditQuestions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>576</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Add/Edit question</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lYear">
     <property name="text">
      <string>Paper Year</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDateEdit" name="deYear">
     <property name="maximumDate">
      <date>
       <year>2500</year>
       <month>12</month>
       <day>31</day>
      </date>
     </property>
     <property name="minimumDate">
      <date>
       <year>2000</year>
       <month>9</month>
       <day>14</day>
      </date>
     </property>
     <property name="displayFormat">
      <string>yyyy</string>
     </property>
     <property name="date">
      <date>
       <year>2023</year>
       <month>9</month>
       <day>14</day>
      </date>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lCompLevel">
     <property name="text">
      <string>Paper Component and level</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="cbComponent"/>
     </item>
     <item>
      <widget class="QComboBox" name="cbLevel"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lQNumber">
     <property name="text">
      <string>Question number</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="sbQNumber">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lQuestionPrompt">
     <property name="toolTip">
      <string>Shown before parts are shown</string>
     </property>
     <property name="statusTip">
      <string>Shown before parts are shown</string>
     </property>
     <property name="text">
      <string>Initial question text</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lBaseMarks">
     <property name="text">
      <string>Marks</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="sbMarks">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pbAccept">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Parts of the question</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QTableWidget" name="twParts">
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Section Number</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Marks</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Contents</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
